bears/general/CLOCBear.py

```diff
@@ -0,0 +1,60 @@
+import re
+
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.results.Result import Result
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from dependency_management.requirements.DistributionRequirement import (
+    DistributionRequirement)
+
+
+@linter(executable='cloc')
+class CLOCBear:
+    """
+    Summarises file of or number of files in directory structure
+    with total lines, number comment lines, actual code lines using tool cloc
+    """
+    LANGUAGES = {'All'}
+    REQUIREMENTS = {DistributionRequirement(apt_get='cloc')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Number of comment lines', 'Number of Empty lines'}
+
+    @staticmethod
+    def create_arguments(filename, file, config_file):
+        return ('', filename)
+
+    def process_output(self, output, filename, file):
+        lines = output.split('\n')
+        files = int(re.search(r'(\d+)', lines[1]).group(1))
+        ignored_files = int(re.search(r'(\d+)', lines[2]).group(1))
+        if(files == ignored_files):
+            msg = 'File should belong to valid programming language.'
+            yield Result.from_values(origin=self,
+                                     message=msg,
+                                     file=filename,
+                                     severity=RESULT_SEVERITY.MAJOR)
```

Comment:
MAJOR seems alright to me, since this is for the condition where no files are valid for the bear.